adding to modules map: Test/heapify2.c -> foo
setting default module to default
note: the -fo-merge-E_variable-and-var-values flag is incompatible with doing module analyses, so turning it off.
Test/heapify2.c:37: warning: goto may require some variables to be free()-ed
Test/heapify2.c:39: warning: break may require some variables to be free()-ed
Test/heapify2.c:40: warning: continue may require some variables to be free()-ed
Test/heapify2.c:42: warning: longjmp may require some variables to be free()-ed

---- START ---- Test/heapify2.c
--- Test/heapify2.c
+++ Test/heapify2.c
@@ -7,4 +7,4 @@
-  int x;
-  &x;
-  int y = 200;
-  &y;
+  int (*x)=(typeof(x))xmalloc2(sizeof *x);
+  &(*x);
+  int (*y) = (typeof(y))xmalloc2(sizeof *y); *y=200;
+  &(*y);
@@ -12,2 +12,2 @@
-    int z;
-    &z;
+    int (*z)=(typeof(z))xmalloc2(sizeof *z);
+    &(*z);
@@ -15,3 +15,3 @@
-      int a = 300;
-      &a;
-      return 1;                 // a, z, y, x
+      int (*a) = (typeof(a))xmalloc2(sizeof *a); *a=300;
+      &(*a);
+      {free2(a);free2(z);free2(y);free2(x);return 1;}                 // a, z, y, x
@@ -19,1 +19,1 @@
-    return 2;                   // z, y, x
+    {free2(z);free2(y);free2(x);return 2;}                   // z, y, x
@@ -21,3 +21,3 @@
-    int b;
-    &b;
-    return 3;                   // b, y, x
+    int (*b)=(typeof(b))xmalloc2(sizeof *b);
+    &(*b);
+    {free2(b);free2(y);free2(x);return 3;}                   // b, y, x
@@ -26,1 +26,1 @@
-    int c;
+    int (*c)=(typeof(c))xmalloc2(sizeof *c);
@@ -28,1 +28,1 @@
-    &c;
+    &(*c);{free2(c);};
@@ -44,1 +44,1 @@
-  return 100;                   // y, x
+  {free2(y);free2(x);return 100;}                   // y, x
---- STOP ----

